kazoo/handlers/eventlet.py

```diff
@@ -1,6 +1,8 @@
 """A eventlet based handler."""
 from __future__ import absolute_import
 
+import atexit
+
```

Comment:
No need for blank line, these are all stdlib imports, right? So can be grouped together in lexicographic order. This comment applies multiple places in the PR.